terasoluna-gfw-functionaltest-web/src/main/resources/i18n/application-messages_ja.properties

```diff
@@ -5,6 +5,7 @@ i.gt.me.0003 = \u6210\u529F\uFF01\uFF01
 i.gt.me.0004 = \u60C5\u5831\uFF01\uFF01
 i.gt.me.0005 = \u3053\u3093\u306B\u3061\u306F\u3000{0}\uFF01\uFF01
 i.gt.me.0006 = <div>\u3053\u3093\u306B\u3061\u306F\u3000\u4E16\u754C\uFF01\uFF01</div>
+i.gt.me.0007 =

```

Comment:
I think this.
If I set the empty string in the message, "<li> </ li>" is to make sure that the output.